apps/taskman/service.py

```diff
@@ -176,6 +186,9 @@ def update_task_status(
                         status=409,
                     )
 
+            if status == TaskStatus.REFINEMENT:
```

Comment:
If I understand the workflow correctly we should auto-close in either Refinement (Published) or In Progress (Review) - I mean in either of these two, not just Refinement.

Reply:
That makes sense, I will submit a fix for that.